src/build.rs

```diff
@@ -29,6 +29,10 @@ use wasm;
 
 use wasm_runtime::RuntimeKind;
 
+const WASM: &str = "wasm32-unknown-unknown";
+const ASM_JS: &str = "asmjs-unknown-emscripten";
+const WASM_EMCC: &str = "wasm32-unknown-emscripten";
```

Comment:
I'm find with making those constants, but please name them the same as the strings, e.g. `WASM32_UNKNOWN_UNKNOWN`